examples/quickstart-diffusion/pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "diffusion_example"
+version = "1.0.0"
+description = "Federated Learning with Diffusion Model and Flower (Quickstart Example)"
+license = "Apache-2.0"
+authors = [
+    { name = "The Flower Authors", email = "hello@flower.ai" },
+    { name = "Aash Mohammad", email = "2810aash.edu@gmail.com" },
+]
+dependencies = [
+    "flwr[simulation]>=1.22.0",
+    "flwr-datasets>=0.5.0",
+    "torch==2.6.0",
+    "torchvision==0.21.0",
+    "torchaudio==2.6.0",
```

Comment:
can we use more recent versions? by the way, do we need `torchaudio` in this example?